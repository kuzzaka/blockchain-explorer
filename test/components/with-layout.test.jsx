import React from 'react';
import { shallow } from 'enzyme';
import WithLayout from '../../components/with-layout';

test('WithLayout snapshot', () => {
  const wrapper = shallow(<WithLayout>markup</WithLayout>);
  expect(wrapper).toMatchSnapshot();
});
