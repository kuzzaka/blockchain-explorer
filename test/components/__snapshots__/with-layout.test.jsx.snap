// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithLayout snapshot 1`] = `
<div
  className="jsx-356941361"
>
  markup
  <JSXStyle
    css="@font-face{font-family:system;font-style:normal;font-weight:300;src:local(\\".SFNSText-Light\\"),local(\\".HelveticaNeueDeskInterface-Light\\"),local(\\".LucidaGrandeUI\\"),local(\\"Ubuntu:Light\\"),local(\\"Segoe UI Light\\"),local(\\"Roboto-Light\\"),local(\\"DroidSans\\"),local(\\"Tahoma\\");}body{font-family:\\"system\\";font-size:18px;}main{margin:0 4rem;}"
    styleId="356941361"
  />
</div>
`;
